Makefile

```diff
@@ -214,7 +214,7 @@ rancherdeployprod: guard-RANCHER_ACCESS_KEY \
                   guard-RANCHER_SECRET_KEY \
                   guard-RANCHER_URL
 	export RANCHER_DEPLOY=true && make composetemplateprod
-	$(call start_service,$(RANCHER_ACCESS_KEY),$(RANCHER_SECRET_KEY),$(RANCHER_URL),prod)
+	$(call start_service,$(RANCHER_ACCESS_KEY_PROD),$(RANCHER_SECRET_KEY_PROD),$(RANCHER_URL_PROD),prod)
```

Comment:
I dont't like the idea to change variable names. But if you do, also change the `guard` values

Reply:
It's a different server, that is why.
`RANCHER_ACCESS_KEY` , `RANCHER_SECRET_KEY`, `RANCHER_URL_PROD`
are the default variables for rancher.
I don't think we want to have variable that deploy to prod per default. I did the same in proxywms.